src/AWS.Deploy.CLI/CredentialProfileStoreChainFactory.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using Amazon.Runtime.CredentialManagement;
+
+namespace AWS.Deploy.CLI
+{
+    /// <summary>
+    /// Represents a factory for creating CredentialProfileStoreChain
```

Comment:
Replace explicit string reference of `CredentialProfileStoreChain` to a `<see cref="CredentialProfileStoreChain">`